README.md

```diff
@@ -2,46 +2,134 @@
 
 source code = website
 
-> Build static websites for software projects with zero (currently a fair amount of) configuration.
+> Build a website for your software projects with one command (and zero configuration).
 
-![Logo](https://d30y9cdsu7xlg0.cloudfront.net/png/43478-200.png)
+```
+landr deploy
+```
+
+## How it works
+
+When you run landr on your local repository, it gathers info by leveraging standard conventions.
+It'll first look for a git remote from `github.com` and retrieve some basic information about your project from the github api (`contribution stats`, `releases`), it will then parse standard files like `README.md`, `CHANGELOG.md`, `CONTRIBUTING.md` & `/docs`, it'll then use the data to build out beautiful pages for your website.
```

Comment:
Can we create a more detailed list of all the conventions landr understands, what are their rules, heuristics, etc?